config-bootstrapper/src/main/resources/configs/config-bootstrapper/attribute-service/hypertrace/hypertrace_span_attributes.conf

```diff
@@ -279,6 +279,16 @@ commands = [
                 source_path: API_BOUNDARY_TYPE
               }
             },
+            {
+              fqn: Span.metrics.span_count,
```

Comment:
use https://github.com/hypertrace/config-bootstrapper/blob/53188be9d16065ff03374f0769803c25f2ce813d/config-bootstrapper/src/main/resources/configs/config-bootstrapper/attribute-service/hypertrace/hypertrace_api_trace_attributes.conf#L195-L204 as your example here. Count is the name of the aggregation, we don't want it as part of the attribute name.

Reply:
resolved